src/Adapter/DoctrineItems.php

```diff
@@ -160,16 +167,17 @@ public function list(int $limit = 10, ?string $cursor = null) : ItemListPage
             ->orderBy('item.sequence')
             ->setMaxResults($limit + 1);
 
-        if ($cursor) {
+        if (is_string($cursor)) {
```

Comment:
consider merging this check with the guards for `$cursor` at the top of the method, here it looks like a blind cast to integer whereas it isn't